bin/wp-config.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
This file is called only from `onchange-db.sh`. That's ok today because the config file is almost entirely generated from environment vars. The exception is the `mysql-primary` found via Consul.

However, if we implement git2consul per https://github.com/autopilotpattern/wordpress/issues/4, we'll want a better way to trigger the wp-config updates.